File: MusicShop.Services/Database/Synthesizer.cs
using System;
using System.Collections.Generic;

namespace MusicShop.Services.Database
{
    public partial class Synthesizer : Product
    {
        public int? KeyboardSize { get; set; }
        public bool? WeighedKeys { get; set; }
        public int? Polyphony { get; set; }
        public int? NumberOfPresets { get; set; }

    }
}

File: UI/musicshop_admin/lib/models/synthesizer/synthesizer.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:musicshop_admin/models/Abstract/product.dart';

part 'synthesizer.g.dart';

@JsonSerializable()
class Synthesizer {
  int? id;
  String? ProductNumber;
  int? ProductImageId;
  int? BrandId;
  String? Model;
  double? Price;
  String? Description;
  DateTime? CreatedAt;
  DateTime? UpdatedAt;
  int? KeyboardSize;
  bool? WeighedKeys;
  int? Polyphony;
  int? NumberOfPresets;

  Synthesizer();

  factory Synthesizer.fromJson(Map<String, dynamic> json) =>
      _$SynthesizerFromJson(json);

  Map<String, dynamic> toJson() => _$SynthesizerToJson(this);
}
